### PR TITLE
docs(core): add changelog render options on manage releases page

### LINKED ISSUE #22809: Document possible changelog render options for default renderer
### Documentation issue

- [ ] Reporting a typo
- [ ] Reporting a documentation bug
- [x] Documentation improvement
- [ ] Documentation feedback

### Is there a specific documentation page you are reporting?

https://nx.dev/features/manage-releases

### Additional context or description

After some digging I discovered that it's possible to tweak the changelog output, like disabling the authors section. Unfortunately this isn't documented.

### PR DESCRIPTION
**Pull request by marcusthelin**
Add a "customize changelog output" section to describe the available render options as described by in [this file](https://github.com/nrwl/nx/blob/master/packages/nx/release/changelog-renderer/index.ts).

## Related Issue(s)
Fixes #22809

### CODE REVIEW

**Comment by marcusthelin** [comment]
Formatting is fixed, so it's ready to be reviewd @juristr

**Review by JamesHenry — CHANGES_REQUESTED** [review]
Thanks @marcusthelin, we are making some changes to the interface of the changelog renderer right now, so we can land some docs once that goes in. I'll revert on here once that is the case

File: docs/shared/features/manage-releases.md [hunk 1]
```diff
@@ -57,6 +57,40 @@ The configuration lives in your `nx.json` file under the `"release"` section.
 }
 ```
 
+### Customize changelog output
+
+Changelog render options can be passed as [an object](https://github.com/nrwl/nx/blob/36f297cf7607fa63112415a0b5d76de09193746c/packages/nx/release/changelog-renderer/index.ts#L48) under `release.changelog.projectChangelogs.renderOptions` and `release.changelog.workspaceChangelog.renderOptions` in your `nx.json` file. Below are all options with their default values for the built-in changelog renderer.
```
Comment (JamesHenry) [inline docs/shared/features/manage-releases.md:62]: Please change this to not point to a specific commit. I appreciate you want it to remain valid, but the reality is this whole section needs to be kept up to date any way in order to remain valuabale

**Review by JamesHenry — CHANGES_REQUESTED** [review]

File: docs/shared/features/manage-releases.md (hunk 1 above)
Comment (marcusthelin) [reply docs/shared/features/manage-releases.md]: Valid point. I also removed the line reference since lines tend to change as well.

**Review by JamesHenry — APPROVED** [review]
Thank you!

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**